cockatrice/src/dlg_settings.cpp

```diff
@@ -649,7 +649,7 @@ void MessagesSettingsPage::retranslateUi()
     chatGroupBox->setTitle(tr("Chat settings"));
     highlightGroupBox->setTitle(tr("Custom alert words"));
     chatMentionCheckBox.setText(tr("Enable chat mentions"));
-    chatMentionCompleterCheckbox.setText(tr("Enable mention completer"));
+    chatMentionCompleterCheckbox.setText(tr("Enable mention mentionCompleter"));
```

Comment:
I feel like you CTRL+F'd and replaced for this, or did you change it for another reason?